frontend/src/components/CreateCollectionModal/components/Content/components/LinkInput/index.tsx

```diff
@@ -140,14 +140,16 @@ const LinkInput: FC<Props> = ({
   }
 };
 
-function isDOILink(
+export function isDOILink(
   type: COLLECTION_LINK_TYPE
 ): (value: string) => true | string {
   return (value: string) => {
     // Skip validation if type is not DOI
     if (type !== COLLECTION_LINK_TYPE.DOI) return true;
-
-    const isValid = value.includes("doi.org");
+    const origin = "doi.org/";
+    const originIndex = value.indexOf("doi.org/");
+    const isValid =
```

Comment:
Maybe regex? e.g. `/doi.org\d+/` (not tested 😆)

Reply:
I'm going to leave it as is for now to keep it simple. If we move to actually validate the DOI itself I'll move to regex

Reply:
sounds good!